.travis.yml

```diff
@@ -21,11 +21,9 @@ jobs:
       after_script:
         - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
     - stage: Build & Publish
-      if: branch = master AND type != pull_request
+      if: branch IN ("master", "beta", "alpha") AND type != pull_request
```

Comment:
`beta` channel is configured by default on `semantic-release`, so running `npx semantic-release` should be enough (See: https://semantic-release.gitbook.io/semantic-release/usage/configuration#branches)

Reply:
ok for "beta".
Is "alpha" really necessary here? Do we plan to create an "alpha" branch?

Reply:
It is not necessary, but we've added it since it has the same pre-release behavior. We will remove it 👍